test/helpers/erc7579-modules.js

```diff
@@ -0,0 +1,17 @@
+const SocialRecoveryExecutorHelper = {
+  RecoveryStatus: {
+    NotStarted: 0,
+    Started: 1,
+    Cancelled: 2,
+  },
+  RECOVERY_MESSAGE_TYPE: {
+    RecoveryMessage: [
+      { name: 'account', type: 'address' },
+      { name: 'nonce', type: 'uint256' },
+    ],
```

Comment:
Similarly, using `const { formatType } = require('@openzeppelin/contracts/test/helpers/eip712');` you can do

```suggestion
    RecoveryMessage: formatType({ account: 'address', nonce: 'uint256' }),
```